api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/redis/BaseRedisProperties.java

```diff
@@ -64,11 +66,18 @@
      */
     private boolean useSsl;
 
+    /**
+     * Setting that describes how Lettuce routes read operations to replica nodes.
+     * see on https://github.com/lettuce-io/lettuce-core/wiki/ReadFrom-Settings
```

Comment:
Remove link, and explain behavior in more detail. Add acceptable values, and what each could mean, briefly.